resources/lang/en/common.php

```diff
@@ -108,18 +108,6 @@
         'target' => 'drop here to upload',
     ],
 
-    'group_name' => [
-        'bot' => 'Automated Message',
```

Comment:
I'm pretty sure that "automated message" won't really work globally, would be better to replace it with something more universal so it doesn't fit on modding discussions (and possibly some other message stuff) only.